bokeh/model/model.py

```diff
@@ -74,9 +76,23 @@ class Model(HasProps, HasDocumentRef, PropertyCallbackManager, EventCallbackMana
 
     '''
 
+
+    # a canonical order for positional args that can be
+    # used for any functions derived from this class
+    _args = ()
+
+    _extra_kws = {}
+
     @classmethod
     def __init_subclass__(cls):
         super().__init_subclass__()
+
+        # in order to add an __init__ signature, the cls has to actually have an __init__ of its own. . .
+        # assert "__init__" in cls.__dict__, str(cls)
```

Comment:
@mattpap is there currently a way to identify built-in models easily? I would like to turn this (and other) assertions on, but they can only apply to models we control directly (i.e. not user extensions).

Reply:
I think the simplest way is to check `__module__` whether it starts with `bokeh.`.